Add files via upload

MrAxel777 opened the pull request.

MrAxel777 committed:
Add files via upload

mrgreyves reviewed (changes requested):
Здравствуйте!)
Рекомендую свериться с методичкой
Не хватает большей части файлов необходимых для выполнения данного дз
Нужно доработать =)

MrAxel777 commented:
Ничего непонятно в методичке. У меня ошибки.

git status
On branch play-travis
Your branch is up to date with 'origin/play-travis'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	new file:   .github/.pre-commit-config.yaml
	new file:   .github/PULL_REQUEST_TEMPLATE.md
	new file:   .github/play-travis/test.py
	new file:   .github/workflows/auto-assign.yml
	new file:   .github/workflows/run-tests.yml
	new file:   .pre-commit-config.yaml
	new file:   3dxPhr5
	new file:   play-travis/test.py

Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   .github/play-travis/test.py
	deleted:    3dxPhr5

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	.github/auto_assign.yml
	.github/get-pip.py
